agreement_legal/__manifest__.py

```diff
@@ -11,7 +11,7 @@
     "website": "https://github.com/OCA/contract",
     "category": "Partner",
     "license": "AGPL-3",
-    "version": "12.0.1.0.0",
+    "version": "12.0.1.0.1",
```

Comment:
```suggestion
    "version": "12.0.1.1.0",
```

The changes require a module upgrade to be effective. A server restart would have no effect:
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers